native/src/modules/layout/containers/HeaderContainer.js

```diff
@@ -51,7 +71,12 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType): StatePropsTy
     !languages || languages.status !== 'ready') {
     // Route does not exist yet. In this case it is not really defined whether we are peek or not because
     // we do not yet know the city of the route.
-    return { language: state.contentLanguage, routeCityModel, peeking: false, categoriesAvailable: false }
+    return {
```

Comment:
I thought we fixed these :see_no_evil: Did we set the wrong option? Could you check if the option set is `chop down if to long`? Thanks

Reply:
We fixed it, this branch is just older ;)